### PR TITLE
v0.3.0

### PR DESCRIPTION
**Pull request by evanweible-wf**
Includes changes from #38, #39, #40, and #41. See CHANGELOG for description of all public changes.

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: pubspec.yaml [hunk 1]
```diff
@@ -1,34 +1,30 @@
 name: codemod
-version: 0.2.0
+version: 0.3.0
```
Comment (greglittlefield-wf) [inline pubspec.yaml:2]: Any reason to not go to 1.0.0? 🙂

**Review by greglittlefield-wf — DISMISSED** [review]
One question, otherwise looks good!

File: pubspec.yaml (hunk 1 above)
Comment (evanweible-wf) [reply pubspec.yaml:2]: I was going to follow this up with a null-safe version, so I was sort of waiting for that

Comment (greglittlefield-wf) [reply pubspec.yaml:2]: Ooh good point; that's fair!

**Review by corwinsheahan-wf — APPROVED** [review]

**Comment by evanweible-wf** [comment]
QA +1
- [x] CI passes
- [x] Verified existing over_react_codemod tests pass when upgraded to this branch

@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**